Vagrantfile

```diff
@@ -6,16 +6,22 @@ require "ipaddr"
 boxname = "elastic/ubuntu-16.04-x86_64"
 base_ip = "192.168.124.100"
 domain_name = "swarm.com"
+default_ram = 2048
+default_cpu = 2
 
-nodes = [
-  {hostname: 'm01',    domain: domain_name,    box: boxname,    memory: 2048,    autostart: true},
-  {hostname: 'm02',    domain: domain_name,    box: boxname,    memory: 2048,    autostart: true},
-  {hostname: 'w01',    domain: domain_name,    box: boxname,    memory: 2048,    autostart: true},
-  {hostname: 'w02',    domain: domain_name,    box: boxname,    memory: 2048,    autostart: true},
-  {hostname: 'w03',    domain: domain_name,    box: boxname,    memory: 2048,    autostart: true},
-  {hostname: 'w04',    domain: domain_name,    box: boxname,    memory: 2048,    autostart: true},
+workers = 3
```

Comment:
What if we let people define those as env vars too without having to edit the Vagrantfile?

Like:

```Vagrantfile
$ git diff
diff --git a/Vagrantfile b/Vagrantfile
index cd22272..fc8689b 100644
--- a/Vagrantfile
+++ b/Vagrantfile
@@ -9,8 +9,8 @@ domain_name = "swarm.com"
 default_ram = 2048
 default_cpu = 2
 
-workers = 3
-masters = 3
+masters = ENV.has_key?('SWARM_MASTERS') ? ENV['SWARM_MASTERS'].to_i : 3
+workers = ENV.has_key?('SWARM_WORKERS') ? ENV['SWARM_WORKERS'].to_i : 3
 
 nodes = []
```